src/modules/create_institution/app/create_institution_usecase.ts

```diff
@@ -0,0 +1,38 @@
+import { randomUUID } from 'crypto';
+import { Institution } from "../../../core/structure/entities/Institution";
+import { MissingParameter } from '../../../core/helpers/errors/ModuleError';
+import { IInstitutionRepo } from '../../../core/repositories/interfaces/IInstitutionRepo';
+
+export class CreateInstitutionUsecase {
+    public database_repo: IInstitutionRepo;
```

Comment:
Here u need to import the IUserRepo too, because u need to check if the user is in our db.